packages/axes/src/inputType/PanInput.ts

```diff
@@ -66,6 +67,15 @@ export const getDirectionByAngle = (
  * - touch: 터치 입력 장치
  * - mouse: 마우스
  * - pointer: 마우스 및 터치</ko>
+ * @param {String[]} [inputKey=[]] Allow input only when one of these keys is pressed. If the array is empty, it accepts all keys with case of no key is pressed.
+ * - shift: shift key
+ * - ctrl: ctrl key
+ * - alt: alt key
+ * - meta: meta key <ko>이 중 하나의 키가 눌린 상태에서만 입력이 허용된다. 배열이 비었다면 아무 키도 눌리지 않은 상태와 모든 키가 눌린 상태 모두 허용한다.
+ * - shift: shift 키
+ * - ctrl: ctrl 키
+ * - alt: alt 키
+ * - meta: meta 키 </ko>
```

Comment:
The action of inputKey is the same as `or`.

If so, does it also support `and' behavior?

Reply:
Currently this does not support `and` behavior. I designed this as `or` for the following reasons.
- Certain input devices do not have an Alt key, so it need a replacement.
- A similar option `inputButton` is implemented as `or` and both options are array, it seems better to work consistently with `or` .

But this doesn't support `and' behavior, also it cannot prevent an empty `inputKey` array that allows input with no key pressed also allows input with holding down the `ctrl` key since this was the old behavior.
I'm thinking about this because some PanInput with the ctrl key to must work separately from the PanInput where `inputKey` is an empty array.

Reply:
I think it can support both `and` and `or` if we use an option type more complex than array. 
I will list some of them and share with additional comment.